cmd/delete/account/account.go

```diff
@@ -0,0 +1,130 @@
+package account
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/config"
+	"github.com/loginradius/lr-cli/request"
+
+	"github.com/spf13/cobra"
+)
+
+var fileName string
+
+var inpEmail string
+var inpUID string
+
+type Result struct {
+	IsDeleted      bool `json:IsDeleted`
+	RecordsDeleted int  `json:RecordsDeleted`
+}
+
+func NewaccountCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "account",
+		Short:   "delete account",
+		Long:    `This commmand deletes account`,
+		Example: heredoc.Doc(`$ lr delete account --email <email> (or) --uid <uid>`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if inpEmail == "" && inpUID != "" {
+				return deletebyUID(inpUID)
+			}
+			if inpUID == "" && inpEmail != "" {
+				return deletebyEmail(inpEmail)
+			}
+			if inpUID == "" && inpEmail == "" {
+				return &cmdutil.FlagError{Err: errors.New("atleast one of the flags is necessary")}
+			}
+
+			return deletebyEmail(inpEmail)
+
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringVarP(&inpEmail, "email", "e", "", "emailID")
+	fl.StringVarP(&inpUID, "uid", "u", "", "emailID")
+
+	return cmd
+}
+
+func deletebyEmail(Email string) error {
+	var resObj *cmdutil.APICred
+	res, err := cmdutil.GetAPICreds()
+	if err == nil {
+		resObj = &cmdutil.APICred{
+			Key:    res.Key,
+			Secret: res.Secret,
+		}
+	} else {
+		resp, err := api.GetSites()
+		if err != nil {
+			return err
+		}
+		resObj = &cmdutil.APICred{
+			Key:    resp.Key,
+			Secret: resp.Secret,
+		}
+		cmdutil.StoreAPICreds(resObj)
+	}
```

Comment:
Instead of repeating this in the same file, we can create one local function and use it right?